frontend/src/pages/SubmitProperty.tsx

```diff
@@ -92,6 +92,7 @@ const SubmitProperty = (props: any) => {
         <MapView
           disableMapFilterBar={true}
           disabled={!!props?.match?.params?.id}
+          showParcelBoundaries={false}
```

Comment:
- Hiding the boundaries by default when submitting a property. This is subject to change after feedback from PO.